tests/detabler/contextmenu.js

```diff
@@ -0,0 +1,71 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: contextmenu,detabler */
+
+(function () {
+	'use strict';
+    bender.editor = {
+        config: {
+            enterMode: CKEDITOR.ENTER_P
+        }
+    };
+    bender.test({
+		setUp: function () {
+			this.command = this.editorBot.editor.getCommand('detable');
+		},
+		contextMenuItemIsPresent: function (editor) {
+			// open and close the context menu, and return true if the detable option was present.
+
+			editor.contextMenu.open(editor.editable());
+
+			var itemExists = false;
+			for (var i = 0; i < editor.contextMenu.items.length; ++i)
+				if (editor.contextMenu.items[i].command == 'detable')
+					itemExists = true;
```

Comment:
Hmm.  Yea, I guess so.  I initially used an approach similar to one of the built in tests where I was counting the number of items in the context menu, but this doesn't make sense when you only have one item to add.